lib/utils/deep-sort-object.js

```diff
@@ -3,7 +3,9 @@ var sortedObject = require('sorted-object')
 
 module.exports = function deepSortObject (obj, sortBy) {
   if (obj == null || typeof obj !== 'object') return obj
-  if (obj instanceof Array) return obj.sort(sortBy)
+  if (obj instanceof Array) return obj.map(function (x) {
```

Comment:
Then nested objects inside arrays would be unsorted. I don't know what purpose the sorting serves in the first place, so this way seems most cautious.